components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ElseImpl.java

```diff
@@ -70,4 +83,240 @@ public Element getSVGString(SVGDocument doc) {
+        Iterator<ActivityInterface> itr = getSubActivities().iterator();
+        int childYTop = yTop;
+        int childXLeft = xLeft + getStartIconWidth() + (getYSpacing() / 2);
+        while (itr.hasNext()) {
+            activity = itr.next();
+            childYTop = centreOfMyLayout - (activity.getDimensions().getHeight() / 2);
+            activity.layout(childXLeft, childYTop);
+            childXLeft += activity.getDimensions().getWidth();
+        }
+
+        setStartIconXLeft(xLeft);
+        setStartIconYTop(yTop);
+        setStartIconTextXLeft(startXLeft + BOX_MARGIN);
+        setStartIconTextYTop(startYTop + BOX_MARGIN + BPEL2SVGFactory.TEXT_ADJUST);
+        getDimensions().setXLeft(startXLeft);
+        getDimensions().setYTop(startYTop);
+
+    }
+
+    @Override
+    public SVGCoordinates getEntryArrowCoords() {
+        int xLeft = 0;
+        int yTop = 0;
+        if (layoutManager.isVerticalLayout()) {
+            xLeft = getStartIconXLeft() + (getStartIconWidth() / 2);
+            yTop = getStartIconYTop();
+        } else {
+            xLeft = getStartIconXLeft() + (getStartIconWidth() / 2);
+            yTop = getStartIconYTop();
+
+        }
+
+        SVGCoordinates coords = new SVGCoordinates(xLeft, yTop);
+
+        return coords;
+    }
+
+    @Override
+    public SVGCoordinates getExitArrowCoords() {
+        SVGCoordinates coords = getStartIconExitArrowCoords();
+
+        if (subActivities != null && subActivities.size() > 0) {
+            ActivityInterface activity = subActivities.get(subActivities.size() - 1);
+            coords = activity.getExitArrowCoords();
+        }
+        return coords;
+    }
+
+
+    protected SVGCoordinates getStartIconExitArrowCoords() {
+        int xLeft = 0;
+        int yTop = 0;
+        if (layoutManager.isVerticalLayout()) {
+            xLeft = getStartIconXLeft() + (getStartIconWidth() / 2);
+            yTop = getStartIconYTop() + getStartIconHeight();
+        } else {
+            xLeft = getStartIconXLeft() + getStartIconWidth();
+            yTop = getStartIconYTop() + (getStartIconHeight() / 2);
+
+        }
+
+        SVGCoordinates coords = new SVGCoordinates(xLeft, yTop);
+
+        return coords;
+    }
+
+
+    protected Element getArrows(SVGDocument doc) {
+        if (subActivities != null) {
+            ActivityInterface prevActivity = null;
+            ActivityInterface activity = null;
+            String id = null;
+            ActivityInterface seqActivity = null;
+            SVGCoordinates myStartCoords = getStartIconExitArrowCoords();
+            SVGCoordinates myExitCoords = getEndIconEntryArrowCoords();
+            SVGCoordinates exitCoords = null;
+            SVGCoordinates activityEntryCoords = null;
+            SVGCoordinates activityExitCoords = null;
+            Iterator<ActivityInterface> itr = subActivities.iterator();
+            Element subGroup = doc.createElementNS("http://www.w3.org/2000/svg", "g");
```

Comment:
String "http://www.w3.org/2000/svg" has been used in several places. Create constants and reference where necessary.